src/proposals/20260115/GroveBase_20260115.sol

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity 0.8.25;
+
+import { Ethereum } from "lib/grove-address-registry/src/Ethereum.sol";
+import { Base }     from "lib/grove-address-registry/src/Base.sol";
+
+import { ForeignController } from "lib/grove-alm-controller/src/ForeignController.sol";
+import { IRateLimits }       from "lib/grove-alm-controller/src/interfaces/IRateLimits.sol";
+import { RateLimitHelpers }  from "lib/grove-alm-controller/src/RateLimitHelpers.sol";
+
+import { ForeignControllerInit, ControllerInstance } from "lib/grove-alm-controller/deploy/ForeignControllerInit.sol";
+
+import { CCTPv2Forwarder } from "lib/xchain-helpers/src/forwarders/CCTPv2Forwarder.sol";
+import { LZForwarder }     from "lib/xchain-helpers/src/forwarders/LZForwarder.sol";
+
+import { CastingHelpers }             from "src/libraries/helpers/CastingHelpers.sol";
+import { GroveLiquidityLayerHelpers } from "src/libraries/helpers/GroveLiquidityLayerHelpers.sol";
+
+import { GrovePayloadBase } from "src/libraries/payloads/GrovePayloadBase.sol";
+
+
+/**
+ * @title  January 15, 2026 Grove Base Proposal
+ * @author Grove Labs
+ */
+contract GroveBase_20260115 is GrovePayloadBase {
+
+    address internal constant GROVE_X_STEAKHOUSE_USDC_MORPHO_VAULT = 0xBeEf2d50B428675a1921bC6bBF4bfb9D8cF1461A;
```

Comment:
This should come from address registry: https://github.com/grove-labs/grove-address-registry/blob/5fd8653a059386b44e4c7f503d5c66547cf90680/src/Base.sol#L58